doc/docs/.vuepress/public/specs/operateurs-api-v3.yaml

```
@@ -415,6 +415,19 @@ components:
               amount:
                 type: number
                 description: montant en centimes d'euros
+    terms_violation_label:
+      type: string
+      enum:
+        - distance_too_short
+        - too_many_trips_by_day
+        - too_close_trips
+        - expired
+      description: |
+        Etiquette possible pour un cas de non respect des CGU détectée :
+          - `distance_too_short` : le trajet a une distance trop faible (<2 km)
+          - `too_many_trips_by_day` : le participant a fait plus de 4 trajets dans la même journée (4 operator_trip_id maximum par usager et par jour)
+          - `too_close_trips` : le trajet est trop rapproché d'un trajet précédent ou suivant réalisé avec la même personne (Délai minimum de 30mn entre 2 trajets conducteur (operator_trip_id) impliquant un même usager [identity_key(operator_trip_id#2_start_datetime - operator_trip_id#1_end_datetime) <= 30mn)
+          - `expired` : le trajet a été envoyé trop tard. [Cf: engagement de délais d'envoi par les opérateurs](https://doc.covoiturage.beta.gouv.fr/bienvenue/faq-foire-aux-questions#quel-est-le-delai-denvoi-des-preuves-de-covoiturage)
```

J'me serai quand même permi d'être plus précis
```suggestion
          - `expired` : le trajet a dépassé la date limite d'envoi de 24h après la date de début du trajet. [Cf: engagement de délais d'envoi par les opérateurs](https://doc.covoiturage.beta.gouv.fr/bienvenue/faq-foire-aux-questions#quel-est-le-delai-denvoi-des-preuves-de-covoiturage)
```